acquisitions/src/main/resources/thunderjet/mod-ebsconet/features/update-ebsconet-order-line-empty-locations.feature

```diff
@@ -0,0 +1,143 @@
+Feature: Update Ebsconet Order Line
```

Comment:
Please add Jira ticket as a comment to easily map this test with the corresponding jira